### PR TITLE
Add counsel-dash

### PR DESCRIPTION
**Pull request by nathankot**
[counsel-dash](https://github.com/nathankot/counsel-dash) is an ivy-mode frontend for Dash docsets :)

### CODE REVIEW

**Comment by nathankot** [comment]
Sorry for the opening/closing, been pinging Travis to re-rerun the build.
There was a missing colon in the `Package-Requires` section, fixed by https://github.com/nathankot/counsel-dash/commit/5ea829311690d7963651e850ff5365f6deccf423 - however re-running the Travis build doesn't seem to pick the change up.

Unless something else is wrong that I am unaware of :(

File: recipes/counsel-dash [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(helm-dash
```
Comment (syohex) [inline recipes/counsel-dash]: `helm-dash` -> `counsel-dash`

File: recipes/counsel-dash [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  :repo "nathankot/counsel-dash"
+  :fetcher github
+  :files ("counsel-dash.el"))
```
Comment (syohex) [inline recipes/counsel-dash]: You need not specify `:files` attribute. It is enough to use default value.

File: recipes/counsel-dash (hunk 1 above)
Comment (nathankot) [inline recipes/counsel-dash]: ahh what was I thinking xD

**Comment by purcell** [comment]
The README has instructions for testing the recipe locally before submitting, BTW... :-) (tl;dr - `C-c C-c`)

**Comment by purcell** [comment]
Merged - thanks. Seems less than optimal to depend on `helm-dash` for an `ivy` helper... perhaps you could push upstream for the core of `helm-dash` to be extracted as a `helm`-independent library...?

**Comment by nathankot** [comment]
😂  thanks @purcell 

I've suggested this on areina/helm-dash#119, just waiting for feedback from the helm-dash maintainers - but I'll be happy to do it if they indicate they will accept such a change :)

**Comment by purcell** [comment]
Super, thanks for following up on that.

**Pull request merged**